personalize.admin.inc

```diff
@@ -336,10 +339,16 @@ function personalize_agent_list($agents = NULL, $allow_edit = TRUE, $allow_alter
   }
 
   $build = array();
+  $build['navigation_bar'] = array(
```

Comment:
Done at [this line](https://github.com/acquia/personalize_module/commit/bad57b3e2ea9cbebdb782ab8938f318509265a5d#diff-336c0df322b81a28385f888327c4f932R443).